NuRadioReco/framework/station.py

```diff
@@ -94,9 +94,36 @@ def iter_channels(self, use_channels=None, sorted=False):
         for channel_id in channel_ids:
             yield self.get_channel(channel_id)
 
+    def iter_trigger_channels(self, use_channels=None):
```

Comment:
docstring? (I know iter_channels doesn't have one either but maybe it helps to distinguish the two?)